packages/cashier/src/stores/on-ramp-store.ts

```diff
@@ -3,23 +3,33 @@ import { localize } from '@deriv/translations';
 import { getKebabCase, isCryptocurrency, routes, websiteUrl } from '@deriv/shared';
 import OnrampProviders from 'Config/on-ramp-providers';
 import BaseStore from './base-store';
+import { TRootStore, TWebSocket, TProviderDetails, TProviderDetailsWithoutFrom } from 'Types';
+
+type TGetDepositAddressResponse = {
+    error: string;
+    cashier: {
+        deposit: {
+            address: string;
+        };
+    };
+};
 
 export default class OnRampStore extends BaseStore {
```

Comment:
@hamid-deriv `disposeThirdPartyJsReaction` and `disposeGetWidgetHtmlReaction` are not defined in the class 🤔 Would you have a look please 🙏🏻

Reply:
@farzin-fs @hamid-deriv `disposeThirdPartyJsReaction` and `disposeGetWidgetHtmlReaction` were created to keep the dispose function returned from mobx `reaction`, and the developer attached them directly to the existing `this` of `OnRampStore` class and used it later in unmounting phase of the component.
And actually, this is what we should always do with the mobx side-effects, to ensure the final cleanup happens.